### ISSUE TITLE
Accessibility name of "power" toggle button in realtime effects dialog is "enable"

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#3492]
### Bug description

I'm assuming that these toggle buttons will be referred to as power toggle buttons in the documentation, and so the accessibility name of the toggle button in realtime effect dialogs should be power, rather than enable.

### Steps to reproduce

Open a realtime effects dialog

### Expected behavior

accessibility name of power toggle button should be "power"

### Actual behavior

It's "enable".

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

### Additional context

See also:
https://github.com/audacity/audacity/issues/3489
https://github.com/audacity/audacity/issues/3482

**Comment by Ryan-Miller-Audacity** [comment]
I tested this and confirmed with NVDA screenreader on windows.

**Comment by Paul-Licameli** [comment]
Will require a re-test to merge #4334

**Comment by Ryan-Miller-Audacity** [comment]
Retested with PR #4334.
